app/global-components.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+
+
+export function BlackButton({ item }: { item: { link: string, text: string | null } }) {
```

Comment:
It would be better to create type `BlackButtonProps` above like so:
```ts
type BlackButtonProps = { 
  item: {
    link: string, 
    text: string | null
  } 
}

export function BlackButton({ item }: BlackButtonProps  ) {...}
```

Reply:
I created a new type and moved this component to a separate file in new folder components